### ISSUE TITLE
Notify users when they have pending invitations

### ISSUE THREAD

**Issue by mconf-daileon** [mconf/mconf-web#454]
---

Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Redmine Issue: 838, http://dev.mconf.org/redmine/issues/838

Original Assignee: Leonardo Daronco

---

Users should receive notifications when they have pending join requests.

After #1067, the spaces will show the user when they have pending invitations, but we should also show these invitations as notifications (or something like it) in other places, specially in the user's home page. It could be a notification in the right sidebar, the same way it's already done to notify private messages.

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Date: 2014-07-02T20:03:18Z

---

Updating this ticket to be more specific, since #1067 already includes most of what was described here previously.

**Comment by mconf-daileon** [comment]
---

Original Redmine Comment
Author Name: **Leonardo Daronco** (Leonardo Daronco)
Original Date: 2014-07-08T14:25:16Z

---

Now all the spaces with a pending join request (invitation or request) are shown in the user's home page, on top of the other spaces, with links to see the invitation/request.
